File: Java/BubbleSort.java
import java.util.Arrays;
import java.util.Scanner;

public class BubbleSort{
  static void bubbleSort(int arr[]){
    for(int i=0;i<arr.length;i++){
      for(int j=0;j<arr.length-i-1;j++){
        if(arr[j]>arr[j+1]){
          int temp =arr[j];
          arr[j]=arr[j+1];
          arr[j+1]=temp;
        }
      }
    }
    System.out.println(Arrays.toString(arr));
  }
    public static void main(String [] args){ 
      System.out.print("Enter Size Of Array => ");
      Scanner sc=new Scanner(System.in);
      int size=sc.nextInt();
        int arr[]= new int[size];
          for(int i=0;i<size;i++){
            arr[i]=sc.nextInt();
          }
          sc.close();
       bubbleSort(arr);
    } 
}

File: Go/BubbleSort.go
package main

import (
	"fmt"
)

func BubbleSort(arr []int) {
	var i, j, temp int
	for i = 0; i < len(arr)-1; i++ {
		for j = 0; j < len(arr)-i-1; j++ {
			if arr[j] > arr[j+1] {
				temp = arr[j]
				arr[j] = arr[j+1]
				arr[j+1] = temp
			}
		}
	}
	fmt.Print(arr)
}
func main() {
	var i, j int
	fmt.Print("Enter Size Of array => ")
	fmt.Scan(&i)
	arr := make([]int, i)
	for j = 0; j < i; j++ {
		fmt.Scan(&arr[j])
	}
	BubbleSort(arr)

}
